.changeset/moody-stingrays-sort.md

```diff
@@ -0,0 +1,6 @@
+---
+'babel-plugin-emotion': minor
+'@emotion/core': minor
+---
+
+`Global` gets handled by the Babel plugin now - this gives inline css-less expressions source maps. The same transforms are being applied for `Global` imported from macro entry (`@emotion/core/macro`).`
```

Comment:
Could you split this changeset into two, one for the babel plugin and one for core?

Reply:
Done